src/hotspot/share/opto/escape.cpp

```diff
@@ -3739,6 +3767,32 @@ void ConnectionGraph::dump(GrowableArray<PointsToNode*>& ptnodes_worklist) {
   }
 }
 
+void ConnectionGraph::print_statistics() {
+  tty->print("No escape = %d, Arg escape = %d, Global escape = %d", Atomic::load(&_no_escape_counter), Atomic::load(&_arg_escape_counter), Atomic::load(&_global_escape_counter));
+  tty->print(" (EA executed in %7.2f seconds)", Atomic::load(&_time_elapsed) * 0.001);
+  tty->print_cr("   ** EA stats might be slightly off since objects might be double counted due to iterative EA **");
```

Comment:
I don't think we need to print this warning. I suggest to put it as comment in this method.